### PR TITLE
Docs ddl named collections

### LINKED ISSUE #46786: Add docs for `CREATE / ALTER / DROP NAMED COLLECTION`
It's a handy new capability added here: https://github.com/ClickHouse/ClickHouse/pull/43252

### PR DESCRIPTION
**Pull request by DanRoscigno**
Adds DDL option to named collection docs

closes https://github.com/ClickHouse/ClickHouse/issues/46786

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: docs/en/operations/named-collections.md [hunk 1]
```diff
@@ -2,34 +2,90 @@
...
-Named collections are stored in the `config.xml` file of the ClickHouse server in the `<named_collections>` section and are applied when ClickHouse starts.
+Parameters set in a named collection can be overridden in SQL, this is shown in the examples
+below.
+
+## Storing named collections in the system database
+
+### DDL example
+
+```sql
+CREATE NAMED COLLECTION name AS
```
Comment (kssenii) [inline docs/en/operations/named-collections.md:28]: I think we should also document `ALTER NAMED COLLECTION .. SET .. DELETE ..` and `DROP NAMED COLLECTION ..`

File: docs/en/operations/named-collections.md [hunk 2]
```diff
@@ -70,6 +69,27 @@ $ cat /etc/clickhouse-server/config.d/named_collections.xml
 </clickhouse>
 ```
 
+## Modifying named collections
+
+Named collections that are created with DDL queries can be altered or dropped with DDL.  Named collections created with XML files can be managed by editing or deleting the corresponding XML.
```
Comment (kssenii) [inline docs/en/operations/named-collections.md:74]: extra space before second sentence

Comment (DanRoscigno) [reply docs/en/operations/named-collections.md]: :) Back in the dark ages, two spaces between sentences were required.

**Comment by DanRoscigno** [comment]
I think that there are named collection examples on hudi and other pages, once this is merged I will find them and link to this page.

**Review by kssenii — APPROVED** [review]

**Pull request merged**